docs/development/develop-prerequisites.md

```diff
@@ -0,0 +1,101 @@
+
+### Git
+
+Setup is described [here](https://git-scm.com/)
+
+### Go
+
+Currently Service Manager requires Go version 1.10. Installation steps can be found [here](https://golang.org/doc/install).
+
+### Dep
+
+Most of the Service Manager github repositories do not include a `vendor` folder. You would need to have `dep` installed and run `dep ensure --vendor-only` to download the project dependencies. Installation details can be found [here](https://github.com/golang/dep#installation).
+
+### GNU Make
+
+Makefiles are provided in the repositories. In order to use them, one would need to setup `make` as described [here](https://www.gnu.org/software/make/manual/make.html).
+
+### Docker
+
+* [Docker for Windows](https://docs.docker.com/docker-for-mac/install/)
+* [Docker for Mac](https://docs.docker.com/docker-for-windows/install/)
+
+### Postgres on Docker
+
+```console
+$ docker run --name postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -d postgres
+```
+
+**Note:** If you are using `Docker for Window` or `Docker for Mac`, the database URI should be `postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable`. Otherwise, instead of `localhost` you would need to use the `$(docker-machine ip)`. 
+
+The obtained value can be used to set the Service Manger `STORAGE_URI`.
+
+## Local K8S Setup
+
+### minikube
+
+A locally running K8S Cluster. Check the [minikube installation guide](https://kubernetes.io/docs/getting-started-guides/minikube/#installation.)
+
+Alternatively, **Windows** users may follow these steps:
+
+> **Note:** On Windows Minikube versions 26.x does not work on Windows. Currently version 25.2 is the latest tested version on Windows that works properly.
+
+* Setup chocolately package manager
+
+```powershell
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+* Install minikube
+
+    ```bat
+    choco install minikube --version 0.25.2
+    ```
+
+>**Note:** In order to reuse the docker daemon and speed up local development check the [Reusing the Docker daemon section of the minikube getting started guide](https://kubernetes.io/docs/getting-started-guides/minikube/#reusing-the-docker-daemon)
+
+### Fulfill the K8S Deployment Prerequisites
+
+The [kubernetes deployment prerequisites]() section outlines some additional steps you would need to perform to fully setup the development environment. 
+
+## Local CF Setup
+
+### PCF Dev
+
+A locally running CF installation that includes an Authorization Server (UAA). Installation steps for PCFDev can be found [here](https://pivotal.io/platform/pcf-tutorials/getting-started-with-pivotal-cloud-foundry-dev/introduction).
+
+#### Install go_buildpack
```

Comment:
https://network.pivotal.io/products/pcfdev#/releases/216556 
PCF dev PAS 2.0.22.0 has buildpack with support for go1.10

Reply:
I removed the buildpack section from this file